client/src/components/Map/index.jsx

```diff
@@ -1,50 +1,40 @@
 import React, { useState, useRef, useCallback } from 'react'
 import {
     GoogleMap,
-    // useLoadScript,
     Marker,
     InfoWindow
 } from '@react-google-maps/api';
 import { features } from '../../utils/dummyData.json';
 import './style.scss';
 import Search from '../SearchInput'
 import FindMeBtn from '../FindMeBtn'
-// import usePlacesAutocomplete, { getGeocode, getLatLng } from 'use-places-autocomplete';
-// import {
-//     Combobox,
-//     ComboboxInput,
-//     ComboboxPopover,
-//     ComboboxList,
-//     ComboboxOption
-// } from "@reach/combobox";
-// import "@reach/combobox/styles.css";
-
 import prkingLogo from './images/mapPic.png'
 import { useStoreContext } from '../../utils/GlobalState';
-// import { UPDATE_MAP_LOCATION } from '../../utils/actions';
+import { useQuery } from '@apollo/react-hooks';
+import { QUERY_ALL_PARKING } from "../../utils/queries";
 
 const containerStyle = {
     width: '80vw',
     height: '70vh'
 };
 // const center = { lat: 37.774, lng: -122.419 }
-const libraries = ['places'];
+// const libraries = ['places'];
 const options = {
     disableDefaultUI: true,
     zoomControl: true
 }
 const plots = features.map(location => (location.geometry.coordinates))
 
-function MyMapComponent() {
-    // const { isLoaded, loadError } = useLoadScript({
-    //     id: 'google-map-script',
-    //     googleMapsApiKey: process.env.REACT_APP_GOOGLE_API,
-    //     libraries
-    // })
+function MyMapComponent(props) {
+
+    // const { loading, data } = useQuery(QUERY_ALL_PARKING);
+
+    // console.log(loading);
+    // console.log(data);
```

Comment:
do we need this unused code?

Reply:
it throws an error at the moment when uncommented but its going to be used to query the parking spots